src/main/java/org/springframework/integration/aws/support/S3Session.java

```diff
@@ -47,21 +47,29 @@
  * @author Artem Bilan
  * @author Jim Krygowski
  * @author Anwar Chirakkattil
+ * @author Xavier FRANCOIS
```

Comment:
I'm not sure that your official name really comes with all letters in uppercase...

No meaning to offense you, but just curios if this is intentional: you have the same in other file, too.

Thanks for understanding.

Reply:
Indeed, it was a mistake, fixed.